
<launch>

    <node pkg="data_logger" name="operator_gui" 
    type="operator_gui.py" output="screen">
 
    </node>

</launch>